### PR TITLE
Cleaned up and added more lockable lockers

### PR DESCRIPTION
**Pull request by Robustin**
Fixes up a lot of the locker prefabs and adds more lockable locker prefabs. 

The prefabs were a mess, my favorite is the Warden's locker was 'locker_HOS2' and it's in-game name was "HoP's locker".

Several prefabs also hadn't been given formal "lockable" treatment. Unfortunately @Ranzarth added lockability directly to the OutpostStation scene in #3355 instead of changing the prefabs and they weren't set up properly. I've added lockability to these prefabs, but there are probably some manually-added locker lights on OutpostStation that will need to be hunted down and removed.

### CODE REVIEW

File: UnityProject/Assets/Resources/Prefabs/Objects/Closets/Closet_Locker_Warden.prefab [hunk 1]
```diff
@@ -301,14 +299,14 @@ MonoBehaviour:
...
+    Melee: 20
+    Bullet: 10
+    Laser: 10
+    Energy: 0
+    Bomb: 10
     Rad: 0
-    Fire: 80
-    Acid: 80
+    Fire: 70
+    Acid: 60
```
Comment (fomalsd) [inline UnityProject/Assets/Resources/Prefabs/Objects/Closets/Closet_Locker_Warden.prefab:309]: did you make these adjustments on purpose?

**Comment by fomalsd** [comment]
some missing references were reported:
```
Missing reference found in scriptable object Closet_Locker_Purple field Object
Missing reference found in scriptable object Closet_Locker_HOS2 field Object

  Expected: <empty>
  But was:  < (Closet_Locker_Purple, Object), (Closet_Locker_HOS2, Object) >at Tests.MissingAssetReferences.CheckMissingRefenceFieldsScritpableObjects () [0x0016a] in /github/workspace/UnityProject/Assets/Tests/Asset/MissingAssetReferences.cs:172[Tests] Missing reference found in scriptable object Closet_Locker_Purple field Object
[Tests] Missing reference found in scriptable object Closet_Locker_HOS2 field Object
```

File: UnityProject/Assets/Resources/Prefabs/Objects/Closets/Closet_Locker_Warden.prefab (hunk 1 above)
Comment (Robustin) [reply UnityProject/Assets/Resources/Prefabs/Objects/Closets/Closet_Locker_Warden.prefab:309]: Weird, no.

**Comment by Robustin** [comment]
@fomalsd 

I'm not familiar with that issue. What's causing that?

**Comment by fomalsd** [comment]
it's our test runner reporting that some fields lost references

**Comment by Robustin** [comment]
Right but everything should update automatically when I changed the name of the prefabs. I'm trying to figure out what would explicitly reference those things. I haven't had any errors on my end.

Does field object mean something existing in the scene?

**Comment by fomalsd** [comment]
just take a look at aforementioned prefabs and see if any fields have Missing references.
missing references don't guarantee that something will immediately break, but develop doesn't have any so they have to be fixed in order to pass the tests. otherwise if merged tests will start failing for everyone

**Comment by ThatDan123** [comment]
In #3428 ive adjusted locker populators.

And if it is merged before your PR, you will need to redo:

Warden
RD
Captain
CMO
CE
Atmos
HOS

**Comment by fomalsd** [comment]
a conflict indeed

**Comment by ThatDan123** [comment]
#3446 changes the CMO locker to the correct prefab. I havnt fixed it not being able to lock as im not sure how that works.

**Comment by Robustin** [comment]
Still trying to figure out where the missing references are too, both of those lockers (Purple, HOS2) had their name changed so I don't know how I could possibly find missing references in something that doesn't exist. I'll see if the sprites or other children have any reference to those old names but I don't quite grasp why Unity wouldn't have updated those references when the name changed.

**Comment by Robustin** [comment]
Ugh resolving conflicts in Unity is a mess, just going to completely redo this PR...

**Pull request closed without merging**